src/main/java/edu/cornell/kfs/krad/antivirus/service/impl/DummyAntiVirusServiceImpl.java

```diff
@@ -12,33 +12,30 @@
 
 public class DummyAntiVirusServiceImpl implements AntiVirusService {
     private static final Logger LOG = LogManager.getLogger();
-    private Status scanResultStatus;
     
-    public DummyAntiVirusServiceImpl() {
-        this(ScanResult.Status.PASSED);
-    }
-    
-    public DummyAntiVirusServiceImpl(Status scanResultStatus) {
```

Comment:
I wonder if this setup could be simplified if we keep the existing constructors and add a String argument constructor that calls this constructor?
That may allow us to leave DummyScanResult and the unit tests as they are for the most part.
Thoughts?

Reply:
I think that these changes make the "dummy" service more closely mimic the actual ClamAV implementation of these services (i.e. reading a value from a properties file and having the ScanResult implementation be responsible for keeping that string "result" value in sync with the "status" enum value) which IMHO is what we want the unit tests to exercise.
I did try your suggestion and ran into issues.